runtime/environment-cpu.yml

```diff
@@ -16,6 +16,9 @@ dependencies:
   - scikit-learn=1.1.2
   - scipy=1.9.1
   - tqdm=4.64.1
+  - interpret=0.3.0
```

Comment:
@Questions1 can you please make sure there is parity between the CPU and GPU versions? I saw you removed `interpret` from `environment-gpu.yml`.

Additionally, it would be nice if you could put these dependencies in alphabetical order with the other ones.